components/Footer.js

```diff
@@ -9,14 +10,31 @@ export default function Footer() {
           'Margarita Humanitarian Foundation is a 501(c)(3) nonprofit registered in the US.'
         }
       </span>
-      <a href="https://vercel.com?utm_source=margaritahumanitarian&utm_campaign=oss">
-        <Image
-          alt="Powered by Vercel"
-          height="44"
-          src="/images/powered-by-vercel.svg"
-          width="212"
-        />
-      </a>
+      <div className="flex justify-center items-center cursor-pointer">
+        <a
+          className="mr-4 mt-2"
+          href="https://vercel.com?utm_source=margaritahumanitarian&utm_campaign=oss"
+        >
+          <Image
+            alt="Powered by Vercel"
+            height="44"
+            src="/images/powered-by-vercel.svg"
+            width="212"
+          />
+        </a>
+        <Link
```

Comment:
why do we use here `Link` if the above link for Vercel works OK with `<a>` tag? It looks like it causes more troubles than profits (do we have any :)) ? The `target="_blank"` seems to not work for `Link` component.

Reply:
I've removed the `Link` and it runs fine on my local machine. It isn't deploying here though.